firefox-ios/firefox-ios-tests/Tests/XCUITests/LoginsTests.swift

```diff
@@ -97,23 +97,28 @@ class LoginTest: BaseTestCase {
         XCTAssertTrue(app.staticTexts[domain].exists)
         // XCTAssertTrue(app.staticTexts[domainLogin].exists)
         XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 1)
-        // Check to see how it works with multiple entries in the list- in this case, two for now
-        app.buttons["Settings"].tap()
-        navigator.nowAt(SettingsScreen)
-        waitForExistence(app.buttons["Done"])
-        app.buttons["Done"].tap()
 
-        navigator.nowAt(HomePanelsScreen)
-        saveLogin(givenUrl: testSecondLoginPage)
-        openLoginsSettings()
-        mozWaitForElementToExist(app.tables["Login List"])
-        XCTAssertTrue(app.staticTexts[domain].exists)
-        // XCTAssertTrue(app.staticTexts[domainSecondLogin].exists)
-        // Workaround for Bitrise specific issue. "vagrant" user is used in Bitrise.
-        if (ProcessInfo.processInfo.environment["HOME"]!).contains(String("vagrant")) {
-            XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 1)
-        } else {
-            XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 2)
+        // iOS 15 may show "Toolbar" instead of "Settings" intermittently.
+        // I can't reproduce the issue manually. The issue occurs only during test automation.
```

Comment:
Intermittent issue: "Toolbar" is supposed to be "Settings". I investigated on `openLoginsSettings()` but could not figure why "Toolbar" would be tapped.
![Simulator Screenshot - iPhone 13 - 2024-05-27 at 17 20 34](https://github.com/mozilla-mobile/firefox-ios/assets/1740517/c1640f73-c31b-41df-9d5d-33a9b61936de)